core/templates/dev/head/header_js_libs.html

```diff
@@ -1,15 +1,11 @@
-<!--[if lt IE 9]>
-  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
-<![endif]-->
-<!--[if gte IE 9]><!-->
-<!-- jquery.js, angular.js and jquery-ui.js are removed from bundled js because they need to be at the header. Including
-bundled js at the header will block rendering.-->
-  <script src="/third_party/static/jquery-2.1.1/jquery.min.js">
-  </script>
-<!--<![endif]-->
+<!-- jquery.js, angular.js and jquery-ui.js are removed from bundled js because
+they need to be at the header. Including bundled js at the header will block
+rendering.-->
+<script src="/third_party/static/jquery-3.0.0/jquery.min.js">
```

Comment:
I suppose not; there's some level of practicality versus pain here. Let's go with what jQuery 3.0.0 needs. Plus, IE 9 final release was 5 years ago.